ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.alp311

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.11
 
 # Install OpenJDK 8 from Alpine because AdoptOpenJDK is 16+, only (February, 2021).
-RUN apk --update add bash shadow openssh-server openssh-client unzip wget openjdk8 git curl make gcc perl xvfb libxrender libxi libxtst procps
+RUN apk --update add bash shadow openssh-server openssh-client unzip wget openjdk8 git curl make gcc musl-dev perl xvfb libxrender libxi libxtst procps m
```

Comment:
Here's an additional m at the end that isn't in the other file. Is that a typo?

Reply:
Yep. Fixed :-) Thanks for the good review!